packages/front-end/components/Features/SDKConnections/SDKConnectionForm.tsx

```diff
@@ -1139,11 +1138,14 @@ export default function SDKConnectionForm({
                     body={
                       <>
                         <p>
-                          Enable the creation &amp; use of ID Lists with over{" "}
-                          {SMALL_GROUP_SIZE_LIMIT} items, which are not
-                          supported by old SDK versions.
+                          Reduce the size of your payload by moving ID List
+                          Saved Groups from inline evaluation to a separate key
+                          in the payload json. Re-using an ID List in multiple
+                          features or experiments will no longer meaningfully
+                          increase the size of your payload.
                         </p>
                         <p>
+                          This feature is not supported by old SDK versions.
```

Comment:
Do we need this text, given it only shows up for new SDKs right? Or is the concern for unversioned and/or related services (remote eval)?

Reply:
Good point, I think the concern there is for unversioned though (or people not realizing they need to bump their package version)

Reply:
That said, I think I'd like to get this merged now and refine the copy more when revisiting this